docs/sources/developers/plugins/metadata.md

```diff
@@ -7,30 +7,178 @@ aliases = ["/docs/grafana/latest/plugins/developing/plugin.json/"]
+| `routes`        | array                   | false    | For data source plugins. Proxy routes used for plugin authentication and adding headers to HTTP requests made by the plugin. For more information, refer to [Authentication for data source plugins](https://grafana.com/docs/grafana/latest/developers/plugins/authentication/). |
+| `sort`          | number                  | false    | Internal property for sorting. Cannot be used as will be overwritten by Grafana.                                                                                                                                                                                                  |
+| `staticRoot`    | string                  | false    |                                                                                                                                                                                                                                                                                   |
+| `streaming`     | boolean                 | false    | For data source plugins. If the plugin supports streaming.                                                                                                                                                                                                                        |
+| `tables`        | boolean                 | false    |                                                                                                                                                                                                                                                                                   |
+| `tracing`       | boolean                 | false    | For data source plugins. If the plugin supports streaming.                                                                                                                                                                                                                        |
+
+## dependencies
+
+Plugin dependencies.
+
+### Properties
+
+| Property         | Type   | Required | Description                                                |
+|------------------|--------|----------|------------------------------------------------------------|
+| `grafanaVersion` | string | **true** | Required Grafana version for this plugin, e.g. `6.7.x`     |
+| `plugins`        | array  | false    | An array of required plugins on which this plugin depends. |
+
+## includes
+
+### Properties
+
+| Property     | Type    | Required | Description |
+|--------------|---------|----------|-------------|
+| `addToNav`   | boolean | false    |             |
+| `component`  | string  | false    |             |
+| `defaultNav` | boolean | false    |             |
+| `name`       | string  | false    |             |
+| `role`       | string  | false    |             |
+| `type`       | string  | false    |             |
+
+## info
```

Comment:
```suggestion
## Info
```

Reply:
Due to a limitation in the generator tool, this is not possible for now. I'll look into improving it later on.